app/views/layouts/admin.html.erb

```diff
@@ -9,6 +9,8 @@
   <%= yield :js %>
   <meta name="viewport" content="width = device-width" />
   <meta name="robots" content="noindex" />
+  <%# theme-color is from app/assets/stylesheets/petitions/_colours.scss:$house-of-commons-green %>
+  <meta name="theme-color" content="#080">
```

Comment:
@willp-bl this only applies to home page style web apps right? No-one is going to be adding the admin site to their phone's home page since it doesn't really work on mobile so you might as well remove it.

Reply:
it will be visible in the normal mobile chrome browser when you look at any admin pages there, same as the screenshots above, let me know if you still want it removed